sdk/storage/storage-blob/tests.yml

```diff
@@ -9,6 +9,12 @@ resources:
       name: Azure/azure-sdk-tools
       endpoint: azure
 jobs:
-  - template: ../archetype-sdk-tests-storage.yml
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-integration.yml
```

Comment:
If no files are using `archetype-sdk-tests-storage` then we should delete that file. However, it looks like the `archetpye-sdk-tests-storage.yml` file is there to test a pretty broad storage matrix. Unless the intent is to narrow that matrix down we should keep using `archetype-sdk-tests-storage.yml`

Reply:
Fixed!